resources/images/s3/filestorage.go

```diff
@@ -112,9 +117,19 @@ func NewSimpleStorageServiceDefaults(bucket, region string) (*SimpleStorageServi
 	}, nil
 }
 
+func getImageByTenant(ctx context.Context, objectID string) string {
+	// If multi-tenancy, prefix objectID with tenant id
+	if id := identity.FromContext(ctx); id != nil && len(id.Tenant) > 0 {
```

Comment:
also consider erroring out here. this code will work fine, but on the off chance that:
- the 'set tenant tag' is set
- there's no 'tenant' in the jwt

maybe it's better to fail.